src/components/noteComponents/MessageLoading.tsx

```
@@ -1,15 +1,60 @@
 'use client';
 
 import Text from '../common/Text';
+import Lottie from 'react-lottie';
+import { useEffect, useState } from 'react';
+import { motion } from 'framer-motion';
 
 const MessageLoading = () => {
+  const [animation, setAnimation] = useState<any>(null);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)